Remotion/Development/Core/NewlineStringExtensions.cs

```diff
@@ -0,0 +1,66 @@
+// SPDX-FileCopyrightText: (c) RUBICON IT GmbH, www.rubicon.eu
+// SPDX-License-Identifier: LGPL-2.1-or-later
+using System;
+using System.Text;
+
+namespace Remotion.Development;
+
+public static class NewlineStringExtensions
+{
+  public static string NormalizeToCurrentOsStyleLineEndings (this string value)
```

Comment:
naming: 
```suggestion
  public static string NormalizeToCurrentOSStyleLineEndings (this string value)
```
2-letter acronym uses upper-case